service-front/module/Application/src/Helpers/LpaStatusTypeHelper.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Helpers;
+
+use Application\Exceptions\LpaTypeException;
+
+class LpaStatusTypeHelper
+{
+    private array $lpaStatusTypes = [
+        'draft' => 'Draft',
+        'in-progress' => 'In progress',
+        'statutory-waiting-period' => 'Statutory waiting period',
+        'registered' => 'Registered',
+        'suspended' => 'Suspended',
+        'do-not-register' => 'Do not register',
+        'expired' => 'Expired',
+        'cannot-register' => 'Cannot register',
+        'cancelled' => 'Cancelled',
+        'de-registered' => 'De-registered',
+    ];
+
+    private array $lpaPermissions = [
+        'donor' => [
+            'draft',
+            'in-progress',
+            'statutory-waiting-period',
+            'do-not-register'
+        ],
+        'certificateProvider' => [
+            'in-progress',
+            'statutory-waiting-period',
+            'do-not-register'
+        ],
+        'voucher' => [
+            'in-progress',
+            'statutory-waiting-period',
+            'do-not-register'
+        ]
+    ];
+
+    private array $personTypes = [
+        'donor' => 'Donor',
+        'certificateProvider' => 'Certificate Provider',
+        'voucher' => 'Voucher'
+    ];
+
+    private bool $startable = true;
+
+    private string $status = 'draft';
```

Comment:
default value not needed - will always be set in `construct`

```suggestion
    private string $status;
```